static/app/views/onboarding/setupDocs.tsx

```diff
@@ -142,27 +152,45 @@ function ProjectDocsReact({
 
   return (
     <Fragment>
-      <SetupIntroduction
-        stepHeaderText={t(
-          'Configure %s SDK',
-          platforms.find(p => p.id === 'javascript-react')?.name ?? ''
-        )}
-        platform="javascript-react"
-      />
-      {productSelectionAssignment === 'variant1' ? (
+      {newOrg ? (
+        <Fragment>
+          <SetupIntroduction
+            stepHeaderText={t(
+              'Configure %s SDK',
+              platforms.find(p => p.id === 'javascript-react')?.name ?? ''
+            )}
+            platform="javascript-react"
+          />
+          {productSelectionAssignment === 'variant1' ? (
+            <ProductSelection
+              defaultSelectedProducts={[
+                PRODUCT.PERFORMANCE_MONITORING,
+                PRODUCT.SESSION_REPLAY,
+              ]}
+            />
+          ) : productSelectionAssignment === 'variant2' ? (
+            <ProductSelection />
+          ) : null}
+        </Fragment>
+      ) : (
         <ProductSelection
```

Comment:
for existing orgs the product selection will come with everything selected by default